Tasks/SonarQubePostTest/PRCA/ReportProcessor-Module.psm1

```diff
@@ -231,4 +233,7 @@ function AnnotateIssuesWithRelativePath
     return $issues
 }
 
-#endregion
\ No newline at end of file
+#endregion
+
+# Export the public functions 
+Export-ModuleMember -Function 'FetchAnnotatedNewIssues'
```

Comment:
One has to explicitly export the functions. This is the equivalent of public / private visibility.